### PR TITLE
Fix some security concerns

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix some security concerns.

### CODE REVIEW

File: client/qiskit_serverless/core/job.py [hunk 1]
```diff
@@ -572,7 +572,7 @@ def filtered_logs(self, **kwargs) -> str:
         """
         return self._job_client.filtered_logs(job_id=self.job_id, **kwargs)
 
-    def result(self, wait=True, cadence=5, verbose=False):
+    def result(self, wait=True, cadence=5, verbose=False, maxwait=0):
```
Comment (Tansito) [inline client/qiskit_serverless/core/job.py:575]: has it sense to just set `wait=False` by default and that's all? I think the final output is similar and it has more sense for me.

**Review by Tansito — APPROVED** [review]
Regardless my comment, I think it has sense a lot of sense these changes @akihikokuroda . Thank you!

File: client/qiskit_serverless/core/job.py (hunk 1 above)
Comment (akihikokuroda) [reply client/qiskit_serverless/core/job.py:575]: The scan suggested that loops needs static termination condition.  So I added not to wait forever.

**Comment by Tansito** [comment]
I'm going to merge this to continue with the release @akihikokuroda , thank you!

**Pull request merged**